src/main/app-updater.ts

```diff
@@ -48,6 +48,8 @@ function handleAutoUpdateBackChannel(event: Electron.IpcMainEvent, ...[arg]: Upd
            * download it from itself via electron.
            */
           electronAutoUpdater.checkForUpdates();
+        } else if (isLinux) {
+          electronAutoUpdater.emit("before-quit-for-update");
```

Comment:
nit: I guess the above comment should apply to this too

Reply:
Sort of, the first line yes but the reason is different.